src/server/controllers/collectives.js

```diff
@@ -28,3 +29,86 @@ export async function info(req, res, next) {
 
   res.send(response);
 }
+
+const query = `
+query Account($slug: String) {
+  account(slug: $slug) {
+    memberOf(role: HOST) {
+      nodes {
+        createdAt
+        account {
+          ... on Collective {
+            slug
+            name
+            website
+            description
+            isApproved
+            balance
+            updatedAt
+          }
+          members(role: [ADMIN,CONTRIBUTOR] ) {
```

Comment:
separate that in 2, get only admins, then get contributors with totalCount.

Maybe getTotalFinancialContributors should be exposed in GraphQL API v2 `stats` to simplify.

Reply:
Exposed `totalFinancialContributors` in GraphQL API v2 `AccountStats`

```js
  totalFinancialContributors: {
        description: 'Total members with role as Backer',
        type: GraphQLInt,
        async resolve(collective) {
          const totalFinancialContributors = await collective.getTopBackers();
          if (isEmpty(totalFinancialContributors)) {
            return 0;
          }
          return totalFinancialContributors.length;
        },
      },
```